tests/GUIUtils.lua

```diff
@@ -99,18 +99,21 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
         { "\27[38;2;120;134;94m", "<120,134,94:>"},
         { "\27[48;2;85;250;33m", "<:85,250,33>"},
         { "\27[38;2;120;134;94;48;2;85;250;33m", "<120,134,94:85,250,33>"},
```

Comment:
Does the Mudlet parser even support that? If not, I don't see why this should. And if it doesn't support that, doesn't sound like it's useful to add support for it either - we haven't heard any requests for this from our users...